charts/unit-tests/templates/_helpers.yaml

```diff
@@ -7,9 +7,13 @@
 {{- end -}}
 
 {{- define "appManager.image" -}}
-{{ include "hf.common.images.image" (dict "imageRoot" .Values.images.appManager "global" .Values.global.huggingface) | quote }}
+{{ include "hf.common.images.image" (dict "imageRoot" .Values.images.appManager "global" .Values.global.huggingface "appVersion" .Chart.AppVersion) | quote }}
```

Comment:
No need to modify this exemple no ?

Reply:
It proves that the .tag field is a fallback value, not used when the tag field is already specified in the image
This way we test the following cases:
Tag field and no fallback specified
No tag field in image but fallback specified 
Both specified
The only one that will break is: none specified (not tested)

Reply:
So I reused one of the existing tests to check this. But if you prefer, we can indeed create an additional unittest